content/en/platform/corda/4.9/community/secure-coding-guidelines.md

```diff
@@ -62,5 +62,5 @@ Make sure your contracts are secure. Check that:
 
 ## Related Content
 Learn more about:
-* [Writing flows](../../../../../en/platform/corda/4.8/enterprise/flow-state-machines.md)
-* [Contracts](../../../../../../en/platform/corda/4.8/enterprise/cordapps/api-contracts.html)
+* [Writing flows](../enterprise/flow-state-machines.md)
```

Comment:
presume this is deliberately still pointing to enterprise?

Reply:
Yeah, no community equivalent.